wallet/bip32_ed25519_wrapper.go

```diff
@@ -32,6 +35,37 @@ const (
 
 type HDPath []uint32
 
+// NewPath constructs and returns a new, default "empty" path, ready to add
+// new accounts to
+// TODO: support multiple chain IDs
+func NewPath() HDPath {
+	hdp, err := StringToHDPath("m/44'/540'/0'/0'")
+	if err != nil {
+		log.Fatalln(err)
+	}
+	return hdp
+}
+
+func (p HDPath) MarshalJSON() ([]byte, error) {
+	return json.Marshal(p.String())
+}
+
+func (p HDPath) UnmarshalJSON(data []byte) error {
+	var aux string
+	err := json.Unmarshal(data, &aux)
+	if err != nil {
+		return err
+	}
+	hdp, err := StringToHDPath(aux)
+	if err != nil {
+		return err
+	}
+	for i := range p {
+		p[i] = hdp[i]
+	}
+	return nil
+}
```

Comment:
You can use a pointer receiver and avoid the copy:

```suggestion
func (p *HDPath) UnmarshalJSON(data []byte) (err error) {
	var aux string
	if err = json.Unmarshal(data, &aux); err != nil {
		return
	}
	*p, err = StringToHDPath(aux)
	return
}
```